src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -344,7 +423,7 @@ public override IEnumerable<string> GetUnchangedPropertyNames()
         /// to the <paramref name="original"/> entity recursively.
         /// </summary>
         /// <param name="original">The entity to be updated.</param>
-        public TStructuralType CopyChangedValues(TStructuralType original)
+        internal void CopyChangedValues(TStructuralType original)
         {
```

Comment:
It's a breaking change? Why back it to internal void?

Reply:
put it back